core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -116,6 +117,10 @@
         config.serviceConfigs().forEach(cfg -> ServiceCallbackInvoker.invokeServiceAdded(cfg, cfg.service()));
     }
 
+    public static ServerBuilder builder() {
```

Comment:
Can you migrate the test code which uses `new ServerBuilder()`
https://github.com/line/armeria/search?q=ServerBuilder&unscoped_q=ServerBuilder

Reply:
Of course, I'll migrate test code !